### PR TITLE
App updates

### PR DESCRIPTION
**Pull request by gordoncc**
- adding talk and groups to tlon, fixing talk/groups shortcode
- adding scooore and %page to apps

### CODE REVIEW

**Comment by thomas-kroes** [comment]
~zod is probably not a good ship to hold them under as that will be transferred to the UF soon. I would just add their distribution ship as I don't expect it to change: ~marnec-dozzod-marzod

**Comment by gordoncc** [comment]
> ~zod is probably not a good ship to hold them under as that will be transferred to the UF soon. I would just add their distribution ship as I don't expect it to change: ~marnec-dozzod-marzod

Updated blog to not have ~zod as the developer.  As for Tlon, for some reason entering Tlon as the developer  doesn't create a link to the Tlon page.  Entering in "~zod" does.  No idea why, I'd have to dig into the code.  It's just an internal thing and doesn't effect any data displayed, so I'll add that as a to do and address in a separate PR later.

**Comment by gordoncc** [comment]
Oh, and isn't the distribution ship for Tlon apps "~sogryp-dister-dozzod-dozzod"?  Has that changed recently?

**Comment by jalehman** [comment]
> ~zod is probably not a good ship to hold them under as that will be transferred to the UF soon. I would just add their distribution ship as I don't expect it to change: ~marnec-dozzod-marzod

Definitely not ~marnec. That's a prerelease distribution ship. Ask Tlon -- @jamesacklin or @zalberico

**Comment by jamesacklin** [comment]
~sogryp-dister-dozzod-dozzod for Talk and Groups
~mister-dister-dozzod-dozzod for Landscape (%garden)

**Comment by zalberico** [comment]
See: https://martiancomputing.substack.com/p/urbit-troubleshooting-basics
```
%garden   | source ship: ~mister-dister-dozzod-dozzod
%webterm  | source ship: ~mister-dister-dozzod-dozzod
%landscape| source ship: ~lander-dister-dozzod-dozzod
%groups   | source ship: ~sogryp-dister-dozzod-dozzod
%talk     | source ship: ~sogryp-dister-dozzod-dozzod
```

**Comment by gordoncc** [comment]
Thanks guys, I think we've got the ships all in order.

File: content/applications/~sogryp-dister-dozzod-dozzod/groups.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ shortcode = "~sogryp-dister-dozzod-dozzod/groups"
 license = "MIT"
 image = "https://bootstrap.urbit.org/icon-groups.svg"
 bgColor = "#EFF0F4"
-developer = "Tlon"
+developer = "~zod"
```
Comment (jalehman) [inline content/applications/~sogryp-dister-dozzod-dozzod/groups.md]: Update per Zach's & James' comments.

File: content/applications/~sogryp-dister-dozzod-dozzod/talk.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ shortcode = "~sogryp-dister-dozzod-dozzod/talk"
 license = "MIT"
 image = "https://bootstrap.urbit.org/icon-talk.svg"
 bgColor = "#2B5DC0"
-developer = "Tlon"
+developer = "~zod"
```
Comment (jalehman) [inline content/applications/~sogryp-dister-dozzod-dozzod/talk.md]: Update per Zach's & James' comments.

**Review by jalehman — CHANGES_REQUESTED** [review]

**Review by jalehman — CHANGES_REQUESTED** [review]
Why is the developer still listed as zod? I see that the distribution shops are correct, so my original comment was misplaced, but zod is not the developer -- Tlon is.

**Comment by gordoncc** [comment]
> Why is the developer still listed as zod? I see that the distribution shops are correct, so my original comment was misplaced, but zod is not the developer -- Tlon is.

Either ~zod or Tlon should work fine.  The original Tlon apps were listed under ~zod.  I'm not sure how the matching code works, but I'm guessing ~zod matches enough of the Tlon moons name.  Changed it to "Tlon".

**Review by jalehman — APPROVED** [review]

**Pull request merged**